### ISSUE TITLE
Text spacing styles cannot be applied

### ISSUE THREAD

**Issue by mkdir-washington-edu** [hypothesis/product-backlog#1633]
From the HighWire Press / McGraw-Hill Accessibility spreadsheet: https://hypothes-is.slack.com/archives/C8TPC8XMK/p1736535764817829

**Highwire Press Reproduction**
1. Navigate to ""Annotation sidebar"" button and activate it.
2. Try to access the page content with text spacing styles turned on.
3. Observe that text spacing does not gets applied to the content.

**Highwire Expected Result**
When user styles are applied, text should be displayed as per the user styles to ensure that users can access the content with ease.	

**Highwire Actual Result**
When user tries to access the page with text spacing styles turned on, the styles does not gets applied to the content such as "Public", "Sign up/Log in", "Annotations", "Page Notes" and so on text present in the "Annotation Sidebar". This resulted in users with low vision and learning impairments finding it difficult to read the text.	

**Highwire Rationale**
Fixed dimensions used for text spacing, line spacing, paragraph spacing by web page authors can not be readable for users with low vision or users with learning impairments. These users have a tendency to browse the web using custom style sheets where spacing is defined to meet their needs. A web page should not override user defined styles and thereby make it possible for users to access the content with ease. If the web page overrides user styles, it might become difficult for users to access the content. 
When user styles are applied, text on a web page should not get cut off vertically, overlap with other piece of text and even not remain fixed. Text should be displayed as per the user styles to ensure that users with learning impairments and low vision users can access the content with ease.

**Highwire Recommendation**
Define the text and container sizes using percent (%), em and rem units. In addition, use relative units for defining CSS line-height, letter-spacing and so on properties and ensure that user's text spacing styles are not overridden.

**WCAG documentation**
https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html
